src/main/eo/org/eolang/math/number.eo

```diff
@@ -68,15 +70,23 @@
       ^.n.as-bytes
       7F-C0-00-00
 
+  # Checking if number is positive-infinity
+  [] > is-positive-infinity
+    eq. > @
```

Comment:
@levBagryansky is it a good idea to create new object `is-positive-infinity`?

Reply:
@includealex Yes it is used in `number.exp` and in `number-tests`. If we need `is-infinite` object we need theese objects too.